cmd/ci-operator-prowgen/README.md

```diff
@@ -0,0 +1,33 @@
+prowgen
+=======
+
+Prowgen is a tool that converts [ci-operator configuration](https://docs.ci.openshift.org/docs/architecture/ci-operator/)
+into [Prowjobs](https://docs.prow.k8s.io/docs/jobs/).
+
+The objetive is to avoid tailored `Prowjobs` and made `ci-operator configuration` the only
+source of truth for jobs created based on configurations inside `openshift/ci-operator/config/`.
```

Comment:
I don't think this is true. Prowgen respects the existing configuration, which means that the users can have some extra configuration in their job yaml and prowgen will keep it. I am +1 in removing that logic though.....

Reply:
True, thanks for reading the README by the way :D